src/test/java/com/wcg/liquibase/integration/FileNameNoSpacesIntegrationTest.java

```diff
@@ -0,0 +1,28 @@
+package com.wcg.liquibase.integration;
+
+import com.google.common.io.CharStreams;
+import com.wcg.liquibase.resolvers.LiquibaseLinterIntegrationTest;
+import com.wcg.liquibase.resolvers.LiquibaseIntegrationTestResolver;
+import liquibase.Contexts;
+import liquibase.Liquibase;
+import liquibase.exception.ChangeLogParseException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(LiquibaseIntegrationTestResolver.class)
+class FileNameNoSpacesIntegrationTest {
+
+    @DisplayName("Should not allow file name with spaces")
+    @Test
+    @LiquibaseLinterIntegrationTest(changeLogFile = "file-name no-spaces.xml", configFile = "file-name-no-spaces.json")
```

Comment:
Would be good to add another test to the same file, proving that it works recursively e.g. the root changelog filename is compliant but it `includes one that isn't.

Reply:
Added test covering this